userinterface/planui.ui

```diff
@@ -111,6 +111,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QPushButton" name="reverseButton">
+        <property name="text">
+         <string>Reverse Route</string>
```

Comment:
"Reverse" should be sufficient, as this button already is in a box called "Current Route". Will save space for the other functions. ;-)